helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -350,13 +350,19 @@ agent:
       memory: 500Mi
 
   # To collect container logs and journald logs, it will run the agent as a root user.
-  # To run it as non root user, uncomment below `securityContext` options.
+  # To run it as non root user, change runAsUser and runAsGroup to non-zero value (e.g. 20000) and change runAsNonRoot to true.
```

Comment:
So this security fix is to allow fluentd to run as non-root, but default is still root? Is that correct?

Reply:
> So this security fix is to allow fluentd to run as non-root, but default is still to run as root? Is that correct?

Yes. The default value should run on any environment. If we change it to run as non-root by default, it will be broken for CRI-O runtime.

> Also, whether or not it's run as root, the default path to write checkpoints is no longer under /var/log, right?

Correct